### DISCUSSION TITLE
Compatibility matrix with k8s

### DISCUSSION THREAD

**Discussion by tomas-balaz** [vectordotdev/vector#18033] [General]
Hey guys!

Can I find a compatibility matrix or all supported k8s versions somewhere in the Docs?

A compatibility matrix could be essential for users to quickly determine whether the Vector is compatible with their specific Kubernetes version, thus ensuring a smooth integration and usage experience.

I will upgrade k8s to 1.26, 1.27, respectively, and I don't know if Vector supports these k8s versions.

**Comment by jszwedko** [comment]
Hi @tomas-balaz ! You can see the versions we test against in CI here: https://github.com/vectordotdev/vector/blob/aa605206baaa6db0506ed0698cfd14847abbb5a9/.github/workflows/k8s_e2e.yml#L126-L166
